pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -165,43 +159,17 @@ def pygrb_add_bestnr_opts(parser):
                         "increase above the threshold")
 
 
-def pygrb_add_missed_injs_input_opt(parser):
-    """Add to parser object the arguments for missed injection file."""
-    if parser is None:
-        parser = argparse.ArgumentParser()
-    parser.add_argument("-m", "--missed-file", action="store",
-                        default=None,
-                        help="Location of the missed injections file.")
+# def pygrb_add_missed_injs_input_opt(parser):
+#     """Add to parser object the arguments for missed injection file."""
+#     if parser is None:
+#         parser = argparse.ArgumentParser()
+#     parser.add_argument("-m", "--missed-file", action="store",
+#                         default=None,
+#                         help="Location of the missed injections file.")
 
 
 # =============================================================================
-# Functions to create appropriate FileLists of veto/segment files
```

Comment:
There's numerous comments of "never invoked outside this file". I would remove this comment every time it appears. Instead either add the `__all__` variable at the end of the module to ensure these functions are not imported when doing `import pygrb_postprocessing_utils`, or use an underscore in the filename `def _build_veto_filelist` suggests an internal function.

Reply:
Good point.  I was tracking these because I think some of these functions may end up disappearing, and I wanted to be sure to keep an eye on them.  I opted for single underscore convention.